### ISSUE TITLE
List tags with item

### ISSUE THREAD

**Issue by stevenbird** [aikuma/aikuma#469]
List the tags of a file with the file (whether it is an item or derivative file)

**Comment by stevenbird** [comment]
For now, just put them anywhere convenient.

**Comment by stevenbird** [comment]
@lisaslyis – can we dispay all speaker ids (first 4 characters) and language codes (3 letter codes) with the other tags please? It doesn't matter if an item requires more vertical space.

**Comment by stevenbird** [comment]
After adding a tag, return to the same item screen with the same derivative file open.

**Comment by lisaslyis** [comment]
Do you mean moving language and speakers to be shown like the other Olac,
custom tags?

Or just increasing vertical space so that all of them can be shown in the
current place?

**Comment by Lingomat** [comment]
"After adding a tag, return to the same item screen"

Just to elaborate, currently you are returned to main screen.

**Comment by stevenbird** [comment]
@lisaslyis don't move first language and first speaker, but list all languages and speakers please.

**Comment by stevenbird** [comment]
```
test recording (eng)
Speakers: dlso, qlao, qawd
Languages: eng, kor, deu
OLAC: discourse_singing
Custom: foo, bar
```

**Comment by stevenbird** [comment]
Resolved in #486.
